packages/components/README.md

```diff
@@ -6,7 +6,7 @@ The React components library of Symphony`s design system
 
 ## Requirements
 
-- Node 12+
+- Node 20+
```

Comment:
When we say required, do we have an engine field in package.json restricting usage to NODE 20+

Reply:
@baphony  ?

Reply:
only a .nvmrc file